30-remote-site-setup.sh

```diff
@@ -66,20 +65,24 @@ echo $REMOTE_HOST_KEY >> $root_ssh_dir/known_hosts
 # Populate the bosco override dir from a Git repo
 GIT_SSH_KEY=/etc/osg/git.key
 [[ -f $GIT_SSH_KEY ]] && export GIT_SSH_COMMAND="ssh -i $GIT_SSH_KEY"
-[[ -z $BOSCO_GIT_ENDPOINT || -z $BOSCO_DIRECTORY ]] || \
+if [[ -z $BOSCO_GIT_ENDPOINT || -z $BOSCO_DIRECTORY ]]; then
```

Comment:
it seems like the logic has reversed here -

looks like it used to be "only run bosco-override-setup.sh if both `BOSCO_GIT_ENDPOINT` and `BOSCO_DIRECTORY` are set;

now it's "run bosco-override-setup.sh if at least one of `BOSCO_GIT_ENDPOINT` or `BOSCO_DIRECTORY` is _unset_"

is this change in the logic intentional?

Reply:
Good catch, fixed